.github/workflows/pr.yml

```diff
@@ -0,0 +1,38 @@
+name: PR
+
+on:
+  pull_request:
+    branches: [ main ]
+
+jobs:
+
+  unit_integration_tests:
+    name: unit and integration tests
+    runs-on: ubuntu-latest
+    container: us.gcr.io/cf-rabbitmq-for-k8s-bunny/rabbitmq-for-kubernetes-ci
+    steps:
+    - name: Check out code into the Go module directory
+      uses: actions/checkout@v2
+    - name: Unit tests
+      run: make unit-tests
+    - name: Integration tests
+      run: make integration-tests
+    - name: Helm chart tests
+      working-directory: charts/rabbitmq
+      run: ./test.sh
+
+  system_tests:
+    name: system tests
+    runs-on: ubuntu-latest
+    steps:
+    - name: Check out code into the Go module directory
+      uses: actions/checkout@v2
+    - name: System tests
+      run: |
+        curl -s "https://raw.githubusercontent.com/kubernetes-sigs/kustomize/master/hack/install_kustomize.sh"  | bash
```

Comment:
Do we need this line anymore? From what I understand, `make install-tools` will install kustomize for us.

Reply:
It doesn't at the moment: https://github.com/rabbitmq/cluster-operator/blob/457a376906325c7ad88901cca42e8c4abe9d4d38/tools/tools.go#L5-L10 doesn't contain `kustomize`.

Including `sigs.k8s.io/kustomize/kustomize/v3` (see https://kubernetes-sigs.github.io/kustomize/installation/source/) didn't work for me out of the box but I can try it out again and spend some more time debugging.